README.md

```diff
@@ -23,6 +23,95 @@ The documentation for various Merge category API's can be found here:
 * [HRIS](https://www.merge.dev/docs/hris/overview/) (Human Resource Information Systems)
 * [Ticketing](https://www.merge.dev/docs/ticketing/overview/)
 
+### Basic Usage
+
+Common to all categories is the Merge sdk `Configuration` object, which defines the authentication properties of 
+your requests. The `apiKey` property is the api key of your Merge account, which can be administered in the 
+[Merge dashboard](https://app.merge.dev/configuration/keys). The `accessToken` property will be the 
+`X-Account-Token` header which identifies which of your customers' data you are requesting. See the following
+examples for calling various category api's:
+
+#### Accounting
+
+```
+let conf_accounting = new Configuration({
```

Comment:
nit: `const` over `let` - immutable by default is good but this is example code so nbd

less nitty - can you use camelCase for these variable names? it's more consistent with most JS in the SDK and what i've seen in the wild